docs/en/flight_controller/radiolink_pix6.md

```diff
@@ -0,0 +1,284 @@
+# RadiolinkPIX6 Flight Controller
+
+<Badge type="tip" text="main (planned for: PX4 v1.17)" />
+
+:::warning
+PX4 does not manufacture this (or any) autopilot.
+Contact the [manufacturer](https://radiolink.com.cn/) for hardware support or compliance issues.
+:::
+
+The autopilot is recommended for commercial systems integration, but is also suitable for academic research and any other use.
+
+![RadiolinkPIX6](http://www.radiolink.com.cn/firmware/wiki/RadiolinkPIX6/RadiolinkPIX6.png)
+
+The Radiolink PIX6 is a high-performance flight controller. Featuring STM32F7 CPU, vibration isolation of IMUs, redundant IMUs, integrated OSD chip, IMU heating, and DShot.
+
+::: info
+This flight controller is [manufacturer supported](../flight_controller/autopilot_manufacturer_supported.md).
+:::
+
+## Quick Summary
+
+- Processor
+  - 32-bit ARM Cortex M7 core with DPFPU - STM32F765VIT6
+  - 216 MHz/512 KB RAM/2 MB Flash
+  - 32-bit IOMCU co-processor - STM32F100
+  - 32KB FRAM - FM25V02A
+  - AT7456E OSD
+- Sensors
+  - Bosh BMI088 IMU (accel, gyro)
+  - InvenSense ICM-42688 IMU (accel, gyro)
+  - SPA06 barometer
+  - IST8310 magnetometer
+- Power
+  - SMBUS/I2C Power Module Inputs (I2C)
+  - voltage and current monitor inputs (Analog)
+- Interfaces
+  - 16 PWM Outputs with independent power rail for external power source
+  - 5x UART serial ports, 2 with HW flow control
+  - Camera Input and Video Output
+  - PPM/SBUS input, DSM/SBUS input
+  - RSSI (PWM or voltage) input
+  - I2C, SPI, 2x CAN, USB
+  - 3.3V and 6.6V ADC inputs
+  - Buzzer and Safety Switch
+  - microSD card
+- Weight and Dimensions:
+  - Weight 80g
+  - Size 94mm x 51.5mm x 14.5mm
+
+## Where to Buy
+
+[Radiolink Amazon](https://www.radiolink.com.cn/pix6_where_to_buy)（International users）
+
+[Radiolink Taobao](https://item.taobao.com/item.htm?spm=a21dvs.23580594.0.0.1d292c1bNMdSqV&ft=t&id=815993357068&skuId=5515756705284)（China Mainland user）
+
+## Connector assignments
+
+### Top View
+
+![Pix6 top view](../../assets/flight_controller/radiolink_pix6/top_view.png)
+
+### Left View
+
+![Pix6 left view](../../assets/flight_controller/radiolink_pix6/left_view.png)
+
+### Right View
+
+![Pix6 right view](../../assets/flight_controller/radiolink_pix6/right_view.png)
+
+### Rear View
+
+![Pix6 rear view](../../assets/flight_controller/radiolink_pix6/rear_view.png)
+
+## Pinouts
+
+Unless noted otherwise all connectors are JST GH.
```

Comment:
Power ports aren't JST GH and that is not marked.

Reply:
Up to you. You could just add a note here like (assuming this is true)

```suggestion
All connectors are JST GH except the power ports, which are HY2.0-6
```

If it is not true - e.g. if you have a USB or something else, I'd add a note above the section for the respective pinouts (and leave this as it is)